Jasmine/Grid/ViewControllers/GridGameViewController.swift

```diff
@@ -156,6 +162,8 @@ fileprivate extension GridGameViewController {
             return
         }
 
+        self.squareGridViewController.update(collectionData: viewModel.gridData)
```

Comment:
Unnecessary if VM calls "updateGridData()" during the swap tile command called by VC.